include/dlaf/lapack_tile.tpp

```diff
@@ -10,6 +10,17 @@
 
 #include "dlaf/common/assert.h"
 
+template <class T, Device device>
+void hegst(int itype, blas::Uplo uplo, const Tile<T, device>& a, const Tile<T, device>& b) {
```

Comment:
Would we want to provide a more self-explaining name and have a more "safe" control logic for this parameter?

I'm thinking of something similar to the other enumeration types used for other BLAS/LAPACK parameters.

If not, I would at least add a `DLAF_ASSERT`/`DLAF_ASSERT_MODERATE` check for validating the input.

Reply:
I would just add a `DLAF_ASSERT` for itype.